### DISCUSSION TITLE
PASIFIC

### DISCUSSION THREAD

**Discussion by KayraMertAydin** [sonsuzus/BilsemPython#74] [Genel]
"PASIFIC" takımına katılmak isteyenler buradan isim gün vb. bilgileri paylaşabilir.
-Kayra Mert Aydın
-OYG 2
-Perşembe 16.30 - 18.00

**Comment by Atakanoyg1** [comment]
merhaba katılmak istiyorum adım Atakan ŞAHİN ÖYG 1 Çarşamba 3. ve 4. dersler

**Reply by KayraMertAydin** [reply]
👍🏻

**Reply by Atakanoyg1** [reply]
kusura bakma ama ben başkabir takımdayım 2 takımda ismim olamaz lütfen ismimi listeden kaldırır mısın

**Comment by Barlasgng** [comment]
merhaba ben de katılabilir miyim ismim Barlas GÜNGÖR öyg 1 çarşamba 1. ve 2. ders

**Reply by KayraMertAydin** [reply]
👍🏻

**Reply by KayraMertAydin** [reply]
adın 2 takımda görünüyor hangi takımdasın

**Comment by sonsuzus** [comment]
@kadirefeyilmaz27
ve Akın Buzoğlu github hesapları şimdilik sorunlu bunları da alır mısın takıma?

**Comment by KayraMertAydin** [comment]
arkadaşlar lütfen telefon numaralarınızı yazabilir misiniz

**Reply by kadirefeyilmaz27** [reply]
05343089349

**Comment by KayraMertAydin** [comment]
arkadaşlar lütfen yazmayanlar telefon numaralarını yazsın iletişim kopukluğu oluyor.

**Comment by eyupbjk** [comment]
merhaba ben epeydir yoktum katıla bilir miyim 
öyg1 cumartesi ilk iki ders
tel 0541 782 9415
